HDSAviz/data_processing.py

```diff
@@ -0,0 +1,197 @@
+""" Tools for reading and processing the sensitivity analysis data files."""
+import os
+
+import pandas as pd
+
+
+def read_file(filename, numrows=None, drop=False, sep=','):
+    """
+    Function reads a file of input parameters or model results
+    and returns a pandas dataframe with its contents.
+    The first line of the input should contain headers
+    corresponding to the column names.
+
+    Parameters:
+    ----------
+    filename  : String with the complete filename, including
+                absolute or relative path.
+    numrows   : Integer number of rows of the file to read.
+                If you don't specify this parameter all rows
+                will be read.
+    drop      : A list of strings indicating which (if any)
+                of the named columns you do not want to include
+                in the resulting dataframe. (ex. ['cats', 'dogs'],
+                default is not to drop any rows)
+    sep       : String indicating the column separator in the
+                file (optional, default = ',').
+
```

Comment:
I like the format of your function descriptors. I haven't looked at swapils yet, but I think this style would be good to copy so we are consistent in format across the board.